Libraries/Components/View/ViewAccessibility.js

```diff
@@ -55,3 +57,16 @@ export type AccessibilityRole =
 
 // This must be kept in sync with the AccessibilityStatesMask in RCTViewManager.m
 export type AccessibilityStates = $ReadOnlyArray<'disabled' | 'selected'>;
+
+// the info associated with an accessibility action
+export type AccessibilityActionInfo = $ReadOnly<{
+  name: string,
```

Comment:
name sounds a little bit generic and quite similar to label which could cause some confusion. could we name this something like "type" or "actionIdentifier"?

also, could we give this a better type? it's pretty unclear that setting "name" here to "activate", "increment", "decrement", "escape" or "magicTap" has special behavior.